content/en/docs/concepts/cluster-administration/networking.md

```diff
@@ -34,170 +34,45 @@ To learn about the Kubernetes networking model, see [here](/docs/concepts/servic
-   * A CNI metaplugin capable of attaching multiple network interfaces to a container, either through its own CNI, or through delegating the job to any of the popular CNI solution like SRI-OV, or Flannel in parallel
-   * A Kubernetes controller capable of centrally managing both VxLAN and VLAN interfaces of all Kubernetes hosts
-   * Another Kubernetes controller extending Kubernetes' Service-based service discovery concept to work over all network interfaces of a Pod
-
-With this toolset DANM is able to provide multiple separated network interfaces, the possibility to use different networking back ends and advanced IPAM features for the pods.
-
-### Flannel
-
-[Flannel](https://github.com/flannel-io/flannel#flannel) is a very simple overlay
-network that satisfies the Kubernetes requirements. Many
-people have reported success with Flannel and Kubernetes.
-
-### Hybridnet
-
-[Hybridnet](https://github.com/alibaba/hybridnet) is an open source CNI plugin designed for hybrid clouds which provides both overlay and underlay networking for containers in one or more clusters. Overlay and underlay containers can run on the same node and have cluster-wide bidirectional network connectivity.
-
-### Jaguar
-
-[Jaguar](https://gitlab.com/sdnlab/jaguar) is an open source solution for Kubernetes's network based on OpenDaylight. Jaguar provides overlay network using vxlan and Jaguar CNIPlugin provides one IP address per pod.
-
-### k-vswitch
-
-[k-vswitch](https://github.com/k-vswitch/k-vswitch) is a simple Kubernetes networking plugin based on [Open vSwitch](https://www.openvswitch.org/). It leverages existing functionality in Open vSwitch to provide a robust networking plugin that is easy-to-operate, performant and secure.
-
-### Knitter
-
-[Knitter](https://github.com/ZTE/Knitter/) is a network solution which supports multiple networking in Kubernetes. It provides the ability of tenant management and network management. Knitter includes a set of end-to-end NFV container networking solutions besides multiple network planes, such as keeping IP address for applications, IP address migration, etc.
-
-### Kube-OVN
-
-[Kube-OVN](https://github.com/alauda/kube-ovn) is an OVN-based kubernetes network fabric for enterprises. With the help of OVN/OVS, it provides some advanced overlay network features like subnet, QoS, static IP allocation, traffic mirroring, gateway, openflow-based network policy and service proxy.
-
-### Kube-router
-
-[Kube-router](https://github.com/cloudnativelabs/kube-router) is a purpose-built networking solution for Kubernetes that aims to provide high performance and operational simplicity. Kube-router provides a Linux [LVS/IPVS](https://www.linuxvirtualserver.org/software/ipvs.html)-based service proxy, a Linux kernel forwarding-based pod-to-pod networking solution with no overlays, and iptables/ipset-based network policy enforcer.
-
-### L2 networks and linux bridging
```

Comment:
It looks that PR codebase doesn't have a item for this section.

Reply:
Correct - if you look at the currently mentioned [nice little tutorial](https://blog.oddbit.com/2014/08/11/four-ways-to-connect-a-docker/), you will see that since this was documented, Docker has added support for connecting containers to the local network using macvlan networks. So I updated the title to "Docker macvlan networks" since I thought that would be more descriptive/ a better one-liner for a cluster-admin looking to connect her containers to the linux network interface.

Reply:
I should probably have mentioned this in the PR description, my apologies